rosa/aws-load-balancer-operator/deploy-aws-lbo.sh

```diff
@@ -82,11 +99,11 @@ spec:
   startingCSV: aws-load-balancer-operator.v1.0.0
 EOF
 
-
-# Configuring egress proxy for AWS Load Balancer Operator
-# oc -n aws-load-balancer-operator create configmap trusted-ca
-# oc -n aws-load-balancer-operator label cm trusted-ca config.openshift.io/inject-trusted-cabundle=true
-# oc -n aws-load-balancer-operator patch subscription aws-load-balancer-operator --type='merge' -p '{"spec":{"config":{"env":[{"name":"TRUSTED_CA_CONFIGMAP_NAME","value":"trusted-ca"}],"volumes":[{"name":"trusted-ca","configMap":{"name":"trusted-ca"}}],"volumeMounts":[{"name":"trusted-ca","mountPath":"/etc/pki/tls/certs/albo-tls-ca-bundle.crt","subPath":"ca-bundle.crt"}]}}}'
-# oc -n aws-load-balancer-operator exec deploy/aws-load-balancer-operator-controller-manager -c manager -- bash -c "ls -l /etc/pki/tls/certs/albo-tls-ca-bundle.crt; printenv TRUSTED_CA_CONFIGMAP_NAME"
-# oc -n aws-load-balancer-operator rollout restart deployment/aws-load-balancer-operator-controller-manager
-# end of Configuring egress proxy section
+if [ ! -z "${HAS_PROXY}" ]; then
```

Comment:
gate this with a environment variable so user can just run `HAS_PROXY=true ./deploy-aws-lbo.sh`